libparsec/crates/platform_pki/src/lib.rs

```diff
@@ -122,7 +122,7 @@ pub struct SignedMessageFromPki {
 pub use errors::SignMessageError;
 pub use platform::sign_message;
 
-pub use shared::{verify_message, Certificate, SignedMessage};
+pub use shared::{verify_message, Certificate, EndCertificate, SignedMessage};
```

Comment:
see my other comment: I'd rather have all PKI certificate have the x509 prefix (this can be done in another PR though ^^)